Local Pods/GitHubAPI/GitHubAPI/V3NotificationSubject.swift

```diff
@@ -24,3 +24,32 @@ public struct V3NotificationSubject: Codable {
     public let url: URL?
 
 }
+
+public extension V3NotificationSubject {
```

Comment:
Nice, I was looking at the `String` extension we use now and it had some limitations when there wouldn't be a `URL` present (invitations). This is a better place. 👍